server/index.test.js

```diff
@@ -0,0 +1,11 @@
+const request = require('supertest');
+const app = require('./index.js');
+
+test('get locations api test', async () => {
+  await request(app)
+    .get('/api/locations?product_id=1')
+    .expect(200)
+    .then(response => {
+      expect(typeof response).toBe('object');
+    });  
+});
```

Comment:
@gigicupp This looks good! Also nice change to use a querystring param!

I think it would be good practice to add more tests for this going forward, although this is a great start to getting set up. I think you should test that you are indeed getting the correct product_id back from the database, maybe test what happens if you ask for a product_id that doesn't exist, maybe also test that the structure of the data you get back is what you want it to be.

Great start to testing!

Reply:
@masoncpott added res.sendStatus(404) on the server. So if a user requests a product that doesn't exist, then they will be getting 404 error, instead of just "me" seeing the console.log error. added this test.